stage0/run.go

```diff
@@ -314,7 +314,7 @@ func preparedWithPrivateUsers(dir string) (string, error) {
 
 // Run mounts the right overlay filesystems and actually runs the prepared
 // pod by exec()ing the stage1 init inside the pod filesystem.
-func Run(cfg RunConfig, dir string) {
+func Run(cfg RunConfig, dir string, dataDir string) {
```

Comment:
As discussed on IRC - this seems reasonable.
Easier way to keep things better will be moving generation/copying into vm of keyfiles from preparation/run time into entering (stage1) but still with using (as You suggested) separate directory (probably `/var/lib/rkt-kvm`).